SingularityBase/src/main/java/com/hubspot/singularity/SingularityRequest.java

```diff
@@ -229,6 +232,10 @@ public ScheduleType getScheduleTypeSafe() {
     return readOnlyGroups;
   }
 
+  public boolean isBounceAfterScale() {
```

Comment:
You need one that is the Json getter that just returns the actual value.

If you want to add a helper, you should @JsonIGnore it.